awx/ui/test/e2e/commands/navigateTo.js

```diff
@@ -5,8 +5,9 @@ exports.command = function navigateTo (url, expectSpinny = true) {
     this.url(url);
 
     if (expectSpinny) {
-        this.waitForElementVisible(spinny);
-        this.waitForElementNotVisible(spinny);
+        this.waitForElementVisible(spinny, () => {
```

Comment:
maybe `waitForSpinny` could be used here? https://github.com/ansible/awx/blob/devel/awx/ui/test/e2e/commands/waitForSpinny.js#L2

Reply:
that function isn't locator-strategy agnostic, I'll see if I can refactor it to default to its usual use, and offer a param here like I did elsewhere.